content/includes/try-it-out-pelican.html

```diff
@@ -0,0 +1,88 @@
+<div class="ui container">
+    <div class="ui horizontal divider">Try it out</div>
+  </div>
+
+  <section id="try-it-out">
+    <div class="ui padded container">
+      <h2 class="ui huge center aligned header">
+        Deploy in 5 minutes and see for yourself
+      </h2>
+
+      <div class="ui very padded centered grid">
+
+        <div class="ten wide computer sixteen wide tablet sixteen wide mobile column">
+          <div class="ui large left aligned basic vertical segment">
+
+            {# Stylized code blocks #}
+            {% call technical.code_block(title="<code>.readthedocs.yaml</code>", icon="fa-file") %}
+            {% markdown %}
+
+            ```yaml
+            version: 2
+            build:
+              os: "ubuntu-22.04"
+              tools:
+                  python: "3"
+              commands:
+                  # Install Pelican and its dependencies
```

Comment:
This file is included in this PR, but not used.

Reply:
He, yeah, I started using Pelican as an example, but then I changed my mind.